src/plugins/menu/_base.scss

```diff
@@ -247,6 +247,9 @@
 
 	#wb-sm {
 		@extend %global-display-block-important;
+		.menu {
+			background: $mb-pnl-body-background;
```

Comment:
@zzucco Can you replicate this fix in this noscript SASS file: https://github.com/wet-boew/wet-boew/blob/master/src/plugins/menu/_noscript.scss